htdocs/core/lib/company.lib.php

```diff
@@ -1052,12 +1052,12 @@ function show_contacts($conf, $langs, $db, $object, $backtopage = '')
     	if (in_array($val['type'], array('date','datetime','timestamp'))) $align.=($align?' ':'').'center';
     	if (in_array($val['type'], array('timestamp'))) $align.=($align?' ':'').'nowrap';
     	if ($key == 'status' || $key == 'statut') $align.=($align?' ':'').'center';
-    	if (! empty($arrayfields['t.'.$key]['checked']))
+    	if (! empty($arrayfields['t.'.$key]['checked']) || ! empty($arrayfields['sc.'.$key]['checked']))
```

Comment:
I don't understand.
If field 't.'.$key was not checked, if you put an || here, you will show the field even if not checked ?

Reply:
![image](https://user-images.githubusercontent.com/3624836/66660377-6604cd00-ec45-11e9-8241-2a9312f93902.png)

Reply:
t.role is never checked